EIPS/blockhash-eip-draft.md

```diff
@@ -4,6 +4,7 @@ title: Add `blockHash` to JSON-RPC methods which accept a default block paramete
 author: Charles Cooper (@charles-cooper)
 type: Standards Track
 category: Interface
+requires: 234
```

Comment:
For consistency and clarity it's usually put at the bottom. See: https://github.com/ethereum/EIPs/blob/master/eip-X.md